docs/tutorials/pre_executed/timeseries.ipynb

```diff
@@ -0,0 +1,1471 @@
+{
```

Comment:
<p>Should we have anything here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>